src/main/java/com/osiris/autoplug/client/tasks/SSHManager.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2021-2022 Osiris-Team.
+ * All rights reserved.
+ *
+ * This software is copyrighted work, licensed under the terms
+ * of the MIT-License. Consult the "LICENSE" file for details.
+ */
+
+package com.osiris.autoplug.client.tasks;
+
+import java.io.IOException;
+
+import com.osiris.autoplug.client.configs.SSHConfig;
+import com.osiris.autoplug.client.network.online.connections.SSHServerConsoleReceive;
+import com.osiris.autoplug.client.network.online.connections.SSHServerSetup;
+import com.osiris.autoplug.client.utils.ConsoleOutputCapturer;
+import com.osiris.jlib.logger.AL;
+
+public class SSHManager {
+    private static Thread sshThread;
+    private static Thread consoleCaptureThread;
+    private static SSHServerSetup sshServerSetup;
+    private static SSHConfig sshConfig;
+    private static ConsoleOutputCapturer capturer;
+
+    static {
+        try {
+            sshConfig = new SSHConfig();
```

Comment:
sshConfig should not be required to be static, call new SSHConfig() every time you need to use it and use it as local variable.